src/android/com/outsystems/firebase/cloudmessaging/build.gradle

```diff
@@ -23,7 +23,7 @@ apply plugin: 'kotlin-kapt'
 dependencies {
     implementation("com.github.outsystems:oscore-android:1.2.0@aar")
     implementation("com.github.outsystems:oscordova-android:1.2.0@aar")
-    implementation("com.github.outsystems:osfirebasemessaging-android:1.1.5@aar")
+    implementation("com.github.outsystems:osfirebasemessaging-android:1.1.7.13@aar")
```

Comment:
should this be `1.1.8`?

Reply:
Yes, I changed it to test something and then forgot to change it back. Thanks!